### PR TITLE
Week 4: Presentation Proposal

### PR DESCRIPTION
**Pull request by VinterSallad**
# Assignment Proposal

## Title

Understanding and Managing Data Drift in Machine Learning with Evidently AI

## Names and KTH ID
  - Alexander Huynh (alehuy@kth.se)
  - Pablo Freire (pablofi@kth.se)

## Deadline

- Week 4

## Category

- Presentation

## Description

We will introduce the concept of data drift in machine learning and why it is a problem in the area of ML. We will then introduce EvidentlyAI, an ML Model monitoring tool which among other things can help detect data drift and also show how you can integrate this tool into your existing projects.

**Relevance**

This is relevant due to the fact that data drift is an important challenge for ML models development, especially for smaller sized teams. Evidently AI proposes a lightweight, open-source solution to manage this issue automatically.

### CODE REVIEW

**Comment by Deee92** [comment]
Hi @VinterSallad, could you summarize what is meant by data drift, and how Evidently AI helps manage it?

**Comment by VinterSallad** [comment]
> Hi @VinterSallad, could you summarize what is meant by data drift, and how Evidently AI helps manage it?

Hi! Datadrift is when the input data drifts away from what the machine learning model was initially trained on, this can degrade the models performance which EvidentlyAI monitors by looking at the input data, and the goal is to alert the user early so that they can stop the model from further degradation.

**Comment by VinterSallad** [comment]
I'm gonna close this pull request because we have had a change of plans, sorry about that!
